DevOps/Communities/Readme.md

```diff
@@ -3,9 +3,9 @@
 - [Community Classroom](https://discord.io/commclassroom)
 - [Eddiehub](http://discord.eddiehub.org/)
 - [MLH](https://discord.gg/mlh)
-- [https://kubesimplify.com/](https://kubesimplify.com/)
+- [Kubesimplify](https://kubesimplify.com/)
```

Comment:
Please add the link as `https://kubesimplify.com/discord`.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix